code/client/makecatalogs

```diff
@@ -268,6 +269,46 @@ def makecatalogs(repopath, options):
                         exit_code = -1
                         continue
 
+            #uninstaller sanity checking
+            do_pkg_check = False
+            uninstaller_type = pkginfo.get('uninstall_method')
+            if uninstaller_type in ['AdobeCCPUninstaller']:
+                do_pkg_check = True
```

Comment:
What's the purpose of defining this variable? Why not just delete lines 276-280?